File: C_Programming/palindrome_number.c
//Program to check whether given number is Palindrome or not.
//A number is palindrome if it is equal to its reverse number, e.g 121, 5555.


#include <stdio.h>

int main()
{	
	long  num;
	
	printf("Enter an integer: ");
	scanf("%ld", &num);

	long original_num = num, reverse_num = 0, remainder;
	
	while(num)
	{
		remainder = num % 10;
		reverse_num = reverse_num * 10 + remainder;
		num /= 10;
	}

	original_num == reverse_num ? printf("\nPalindrome") : printf("\nNot a Palindrome");

	return 0;
}

File: Python/palindrome_number.py
#Program to check whether given number is Palindrome or not.
#A number is palindrome if it is equal to its reverse number, e.g 121, 5555.

num = input("Enter an integer: ")
num = str(int(num))

if num == num[::-1]:
	print("Palindrome")
else:
	print("Not a Palindrome")
	
